server/src/weaverbird/backends/mongo_translator/steps/aggregate.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Union
+
+from weaverbird.pipeline.steps import AggregateStep
+
+
+def translate_aggregate(step: AggregateStep) -> List[Dict[str, Any]]:
+    idblock = {col: f'${col}' for col in step.on}
+    group: Dict[str, Dict] = {}
+    project: Dict[str, Union[str, int]] = {}
```

Comment:
Maybe that type will work for this specific step, but `$project` in general  can take other forms than just `str` or `int`. It can hold other `dict`s, for nester fields or expressions (like `$cond`, `$substr`, etc.).
We may be more safe with just hinting `Dict[str, Any]` here (that's what it as in the front-end version)

Reply:
:+1: